extra-modules/extra-pluralkit/src/main/kotlin/com/kotlindiscord/kord/extensions/modules/extra/pluralkit/PKExtension.kt

```diff
@@ -266,7 +277,7 @@ class PKExtension : Extension() {
 
         event<MessageUpdateEvent> {
             action {
-                val guild = event.channel.asChannelOfOrNull<GuildChannel>()?.getGuild()
+                val guild = event.channel.asChannelOfOrNull<GuildChannel>()?.getGuildOrNull()
```

Comment:
This should never return null unless the guild isn't in the cache - have you actually run into this one?

Reply:
I've not run into it yet, though I'd personally prefer to cover this base in the event of it cropping up in the future